scripts/exregional_run_analysis.sh

```diff
@@ -871,25 +872,19 @@ if [ ${DO_RADDA} == "TRUE" ]; then
       cp_vrfy ${FIX_GSI}/rrfs.starting_satbias_pc ./satbias_pc
     fi
 
-    # radstat (only for EnVar)
-    if [ ${DO_ENS_RADDA} != "TRUE" ]; then	
-      if [ -r ${FIX_GSI}/rrfs.starting_radstat ]; then
-        echo "using satelite radstat files from ${FIX_GSI}"     
-        cp_vrfy ${FIX_GSI}/rrfs.starting_radstat ./radstat.rrfs
-      fi
-    fi  
-	
   fi
 
-  listdiag=`tar xvf radstat.rrfs | cut -d' ' -f2 | grep _ges`
-  for type in $listdiag; do
-    diag_file=`echo $type | cut -d',' -f1`
-    fname=`echo $diag_file | cut -d'.' -f1`
-    date=`echo $diag_file | cut -d'.' -f2`
-    gunzip $diag_file
-    fnameanl=$(echo $fname|sed 's/_ges//g')
-    mv $fname.$date* $fnameanl
-  done
+  if [ -r radstat.rrfs ]; then
```

Comment:
@LiaofanLin-NOAA check whether lines 874-881 are still needed for ensemble radiance DA?

Reply:
@guoqing-noaa The edits look good to me.  The current radstat.rrfs in the $FIX_GSI is in the format of binary.  When netcdf_diag=.true., using this radstat.rrfs starting file could cause issues in radiance DA for both EnVar and EnKF.

Reply:
@guoqing-noaa  To understand the impact with and without radstat (containing satellite diag files) in the GSI-Observer and EnKF, I performed two EnKF cycling experiments.  To summarize, I found that radstat has no impact in EnKF.

Case WO_radstat (without radstat) uses no radstat file at all.  It does not archive satellite diag files after running the GSI-observer.  It does not use any radstat file in the GSI-observer either.  In contrast, Case W_radstat (with radstat) starts to archive radstat files from the 2nd cycle (i.e., the first and spin-up cycle does not perform EnKF).  This radstat will be used in the GSI-Observer of the 3rd cycle.  Both cases use cycling satbias files.  I started the experiments at 00UTC.  Thus, at 02UTC, it is the first time that the GSI-observer uses the previous radstat.  Here are some findings:

1. Within the working folders of the ensmean GSI-observer at 02UTC:

- Case W_radstat has satellite diag files from 01UTC, while Case WO_radstat does not. At 01 and 02 UTC, both ABI-G16 and AMSUA-N19 satellites are available and assimilated.
- The stdout in Case W_radstat has the following printout, while that in Case WO_radstat does not.  It indicates that the GSI-Observer of Case W_radstat indeed reads those satellite diag files from 01UTC in.  

```
INIT_PREDX: Task 1 processing diag_abi_g16 data date= 2022072101
INIT_PREDX: Task 28 processing diag_amsua_n19 data date= 2022072101
```

- I compared 1) satbias_in versus satbias_out within Case W_radstat and 2) satbias_pc versus satbias_pc.out.  They are different, indicating that satellite bias correction is doing something.  Then, I compared satbias_out between two cases.  I compared satbias_pc.out between two cases.  They are the same between the two cases.  This suggests that adding radstat does not affect the satellite bias correction coefficient outputs in the GSI-Observer.  


2. Within the working folders of task run_enkfupdt (EnKF) at 02UTC, the stdout of both cases has the same ensemble mean analysis increment: 

```
ens. mean anal. increment min/max u -14.3805913925 14.5597715378
ens. mean anal. increment min/max v -11.1093559265 18.8727207184
ens. mean anal. increment min/max t -7.52867841721 8.59306240082
ens. mean anal. increment min/max oz -0.108128141107E-06 0.120032765949E-06
ens. mean anal. increment min/max q -0.618271052837 0.632195830345
ens. mean anal. increment min/max ps -2.35135507584 2.89799404144
```


3. At 02UTC, I compared the fv_core.res.tile1.nc and fv_tracer.res.tile1.nc under STMP, randomly from member 02 (i.e., $CASE/$DATE/mem0002/fcst_fv3lam/INPUT).  These two files are the EnKF analyses at 02 UTC.  By using ncdiff, I found no difference between Cases W_radstat and WO_radstat. I did the same comparison for these files at 04UTC.  Again, there is no difference in these files between the two cases.  


4. I had conversations with Haixia Liu and Emily Liu at EMC.  They work on the radiance DA development in GDAS.  They told me that radstat files are not used in GDAS-EnKF.